script.js

```diff
@@ -3,6 +3,7 @@ startBtn.addEventListener("click", playGame);
 const tiles = document.querySelectorAll(".tile");
 let gameState = ["", "", "", "", "", "", "", "", ""];
 let currentPlayer = "x";
+let turns;
```

Comment:
It's not directly related to this story - but we've now reached a point where we have these three global variables that are used to track the various states of the game. I would prefer if these three were combined into a single object that contains all of the data you need to track globally. 

Something along these lines:

```js
let gameData = {
    gameState: ["", "", "", "", "", "", "", "", ""],
    currentPlayer: "x",
    turns: 0
}
```

This way we bundle all of the related data into a single place (kinda like a single source of truth).

It would mean updating all references to gameState, currentPlayer and turns to point to the new object - but I think it would be worth while!

Reply:
Put everything inside gameData and adjusted object variables.